src/components/Grade/Grade.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import cx from 'classnames';
+import classes from './grade.module.scss';
+import Text from '../Text/Text';
+/*
+ * @param type   "A" | "B" | "C" | "D" | "F"
+ */
+const Grade = ({ type, children, className }) => {
+    return <Text type="regular" className={cx(className, classes[type])}> {children}</Text>;
+};
+
+export default Grade;
```

Comment:
I like the idea here but can you use the Grade component I already built?

pull the repo and import it from 'components/Grade'

it deals with the fact that the text isn't perfectly centered when there's an "A+" or "A-" instead of just "A"